components/demo.vue

```diff
@@ -0,0 +1,60 @@
+<script>
+import { mapMutations } from 'vuex'
+
+export default {
+  name: 'LandingPage',
+  data() {
+    return {
+      
+    }
+  },
+  computed: {
+    todos() {
+      return this.$store?.state?.todos?.list || []
+    }
+  },
+  methods: {
+    addTodo (e) {
+      this.$store.commit('todos/add', e.target.value)
+      e.target.value = ''
+    },
+    ...mapMutations({
+      toggle: 'todos/toggle'
+    })
+  }
+
+}
+</script>
+<template>
+  <div>
+    <NuxtLink :to="switchLocalePath('en')" class="lang">English</NuxtLink>
+    <NuxtLink :to="switchLocalePath('fr')" class="lang">Français</NuxtLink>
```

Comment:
Can we change language without using different paths for the routes (i.e. `/fr` or `/en`)?